src/main/java/com/google/error_analyzer/backend/LogDaoHelper.java

```diff
@@ -65,4 +66,37 @@
     public static String getErrorIndexName (String fileName) {
         return fileName.concat("error");
     } 
+
+    //returns index name for a given file name
+    public static String getIndexName(HttpServletRequest request,
```

Comment:
Is LogDaoHelper the best place to keep these functions?
Should we create another helper class?